src/components/Tables/Table.cy.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import Table from "./Table";
+
+describe("<Table />", () => {
+    const data = [
+        {
+            full_name: "Tom",
+            phone_number: 123456,
+        },
+        {
+            full_name: "Sam",
+            phone_number: 999,
+        },
+        {
+            full_name: "Harper Garrett",
+            phone_number: 2171786554,
+        },
+        {
+            full_name: "Adrian Key",
+            phone_number: 3650099130,
+        },
+        {
+            full_name: "Harrell Wallace",
+            phone_number: 4650047935,
+        },
+        {
+            full_name: "Oneill Curtis",
+            phone_number: 7058491995,
+        },
+        {
+            full_name: "Herring Rutledge",
+            phone_number: 1440882899,
+        },
+        {
+            full_name: "Eloise Rowland",
+            phone_number: 2580325390,
+        },
+        {
+            full_name: "Cathryn Burks",
+            phone_number: 7136166489,
+        },
+        {
+            full_name: "Paopao",
+            phone_number: 7136166469,
+        },
+        {
+            full_name: "Forbes Doyle",
+            phone_number: 1377097191,
+        },
+        {
+            full_name: "Agnes Rosales",
+            phone_number: 3334796379,
+        },
+        {
+            full_name: "Jan Orr",
+            phone_number: 1526538148,
+        },
+        {
+            full_name: "Colleen Lowery",
+            phone_number: 3980156139,
+        },
+        {
+            full_name: "Chloe",
+            phone_number: 4567894522,
+        },
+    ];
+
+    const headers = {
+        full_name: "Name",
+        phone_number: "Phone Number",
+    };
+
+    it("renders", () => {
+        cy.mount(<Table data={data} headers={headers} />);
+    });
+
+    it("display is correct", () => {
```

Comment:
I don't think the name of the test matches what it does - perhaps you can make the data only contain the 2 records that you are testing? 

(Since the test currently doesn't ensure the display is correct even if it is passed)

Reply:
We have edited it in our newer version